src/screens/manage-account/manage-account.tsx

```diff
@@ -48,7 +49,7 @@ const mapStateToProps = (state: IReduxState) => {
 
 export const navigationOptions = ({ navigation, theme }: any) => ({
     title: translate('Account.manageAccount'),
-    headerRight: (
+    headerRight: navigation.state.params.blockchain === Blockchain.ETHEREUM && (
```

Comment:
if there are 2 blockchains with tokens this will not work
Add an util function that determines if the blockchain has only native tokens

Reply:
True, we have to config this from `core/blockchain/ethereum/config.ts`

Reply:
@NovacAndrei  is this fixed in the last commit?

Reply:
No, I left a TODO there